aries-site/src/layouts/navigation/Search.js

```diff
@@ -131,22 +173,49 @@ export const Search = ({ setOpen }) => {
   const onEnter = () => {
     if (value) {
       if (suggestions.length === 1) {
-        router.push(nameToPath(suggestions[0].value.title));
+        router.push({
+          pathname: nameToPath(suggestions[0].value.title),
+          query: value && `q=${value}`,
```

Comment:
```suggestion
          query: `q=${value}`,
```
Do we need to check for `value` since it's already in a block `if (value)` ?

Reply:
done